In house patch created in Oracle
Fix of "implicit declaration" of memset()
Will be sent to community.

--- a/avahi-core/fdutil.c	2016-04-20 16:29:44.771963318 -0700
+++ b/avahi-core/fdutil.c	2016-04-20 16:30:28.719264628 -0700
@@ -21,6 +21,7 @@
 #include <config.h>
 #endif
 
+#include <string.h>
 #include <sys/select.h>
 #include <unistd.h>
 #include <fcntl.h>
